tests/transforms/test_mitigate.py

```diff
@@ -537,7 +556,8 @@ def test_diffability_autograd(self):
 
     @pytest.mark.jax
```

Comment:
@natestemen When you add test that uses a specific framework you need to decorate with the right ML framework tag. The four tags are:
```
@pytest.mark.autograd
@pytest.mark.tf
@pytest.mark.torch
@pytest.mark.jax
```

Reply:
I hopefully shouldn't need this though, right? I don't think we want this functionality to only work for a subset of the frame works you listed, do we?

Reply:
Generally we create a single test for each framework as it is easier to set up rather than have to cast parameters to each framework.

Reply:
Sorry, I need to take a step back. I don't think I'm adding any tests that are for a specific framework. At least that is the goal. Do I need to take any action here?

Reply:
Yes now that it works with all framework you can unit test it. For example create a  test `@pytest.mark.torch` `def test_extrapolate_exponential_torch` then inside this test you add a unit test that uses torch tensors. Repeat for the other frameworks. You can take a look in the test suite, you can check tests with those markers in the test suite. It should help a lot

Reply:
Added some further unit tests in https://github.com/PennyLaneAI/pennylane/pull/5972/commits/5248c02227f6672c64c6d420abb8c5108e29f107.